### ISSUE TITLE
html tag 

### ISSUE THREAD

**Issue by yuedingwangji** [bordaigorl/sublime-evernote#171]
when  I use sublime   to  write  something  and send it to evernote  ,it complained that "html tag NF not  allowd in evernote notes" 
such as  when i  write 
(for i=1;i<NF;i++) 
for my test , it seems to think <NF is a uncompletey html tag ,so it can not allowed  to update

**Comment by bordaigorl** [comment]
Yes, as it stands `<NF` may be interpreted as an unfinished tag. (it may be completed by a later `>` in your code).
If your intention is to put `<` in your text you can escape it with `&lt;`.
If your intention is to write verbatim code you have two options:

1. if you only want a note with the code, try the [`Clip to new note`](https://github.com/bordaigorl/sublime-evernote/wiki/Available-Commands#clip-as-new-note) command
2. mark the code as "code" using Markdown's syntax, either inline `` `like this` `` or as a block with a line containing just ```` ``` ```` marking the end and the beginning of the block.

**Comment by yuedingwangji** [comment]
thanks ， I have found that I can write like this :
(for i=0;i < NF;i++)
